styles/style.css

```diff
@@ -67,11 +68,13 @@ button {
 /* Jumbotron */
 
 .jumbotron {
-  background-image: url("../images/header.jpg");
+  background-image: url(/images/header-bike.jpg);
   border-radius: 0;
```

Comment:
The picture didn't show for me here, I think because the folder `images` is in a different directory to the file `style.css` (It is in the same directory as its parent folder `styles` however), it needs the `../` at the start.